tests/web/test_transactions.py

```diff
@@ -44,9 +44,6 @@ def test_post_create_transaction_endpoint(b, client):
 
     assert res.status_code == 202
 
-    assert '../statuses?transaction_id={}'.format(tx.id) in \
-        res.headers['Location']
```

Comment:
👍  Just a side note. This module is yet to be cleaned and integrated with tendermint compaitable apis.